src/Npgsql/NpgsqlSlimDataSourceBuilder.cs

```diff
@@ -477,6 +477,39 @@ public NpgsqlSlimDataSourceBuilder EnableIntegratedSecurity()
         return this;
     }
 
+    /// <summary>
+    /// Sets up network mappings. This allows mapping PhysicalAddress, IPAddress, NpgsqlInet and NpgsqlCidr types
+    /// to PostgreSQL <c>macaddr</c>, <c>macaddr8</c>, <c>inet</c> and <c>cidr</c> types.
+    /// </summary>
+    /// <returns>The same builder instance so that multiple calls can be chained.</returns>
+    public NpgsqlSlimDataSourceBuilder EnableNetworkTypes()
```

Comment:
Both EnableNetwork and Enable{Geometric,Geometry} would read pretty confusing so I suffixed these with Types. As we have done with EnableUnmappedTypes.

Reply:
Yeah, agreed - the Types suffix is good.